vnext/Microsoft.ReactNative/ReactPropertyBag.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "ReactPropertyBag.g.h"
+
+namespace winrt::Microsoft::ReactNative::implementation {
+
+struct ReactPropertyBag : ReactPropertyBagT<ReactPropertyBag> {
+  ReactPropertyBag() = default;
+
+  static Microsoft::ReactNative::IReactPropertyNamespace GlobalNamespace() noexcept;
+  static Microsoft::ReactNative::IReactPropertyNamespace GetNamespace(hstring const &namespaceName) noexcept;
+  static Microsoft::ReactNative::IReactPropertyName GetName(
+      Microsoft::ReactNative::IReactPropertyNamespace const &ns,
+      hstring const &localName) noexcept;
+  Windows::Foundation::IInspectable GetProperty(
```

Comment:
>Windows::Foundation::IInspectable [](start = 2, length = 33)

Are there any property bag usage scenarios that allow us to gauge chattiness? Could the use of IInspectable here entail frequent boxing/unboxing? If so, would an IJSValueReader be a better representation for property values?

Reply:
I do not expect it to be too chatty. We also consider to use the IJSValueReader/IJSValueWriter for some non-WinRT types. Today the non-WinRT types are simply prohibited and give compilation error.